searches/views.py

```diff
@@ -42,7 +42,17 @@ def get(self, request, query, sem_name, year):
         #     course_match_objs = apps.get_app_config('searches').searcher.vectorized_search(request.subdomain, query, sem)[:4]
         # else:
         #     course_match_objs = baseline_search(request.subdomain, query, sem)[:4]
-        course_match_objs = baseline_search(request.subdomain, query, sem).distinct()[:4]
+        # sort by course number
+        def course_comparator(x, y):
```

Comment:
Use better var names than x and y
Add a comment about what you're doing
Could also maybe make this one line with a python ternary operator